src/index.tsx

```diff
@@ -6,14 +6,24 @@ import { BrowserRouter } from 'react-router-dom';
 import CssBaseline from '@material-ui/core/CssBaseline';
 import { createMuiTheme, ThemeProvider } from '@material-ui/core/styles';
 
-const theme = createMuiTheme({
+export const theme = createMuiTheme({
```

Comment:
```suggestion
const theme = createMuiTheme({
```

Theme skal ikke eksporteres, det skal eies av `ThemeProvider` konteksten 

Se `<ThemeProvider theme={theme}> .. </ThemeProvider>` i render-funksjonen